<?xml version="1.0"?>
<launch>
  <arg name="x" default="0" />
  <arg name="y" default="0" />
  <arg name="z" default="0.5" />
  <arg name="yaw" default="0.5" />
  <arg name="rate" default="30" />
  <arg name="radius" default="0.5" />
  <arg name="enable_tracking" default="true" />
  <arg name="round_period"  default="4.0" />
  <arg name="trajectory_style" default="14" />

  <node name="trajectory_generate" pkg="trajectory_generate" type="trajectory_generate" output="screen">
    <param name="x" value="$(arg x)" />
    <param name="y" value="$(arg y)" />
    <param name="z" value="$(arg z)" />
    <param name="yaw" value="$(arg yaw)" />
    <param name="rate" value="$(arg rate)" />
    <param name="radius" value="$(arg radius)" />
    <param name="enable_tracking" value="$(arg enable_tracking)" />
    <param name="round_period"     value="$(arg round_period)" />
    <param name="trajectory_style" value="$(arg trajectory_style)" />
  </node>


</launch>
